io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
```

Comment:
this app-id doesn't meet the requirements. github based id always include the user/org name AND the app name (repository). They have 4 components.

Reply:
From the Flatpak documentation

"io.github.Foo

This is problematic because while foo.github.io may be unique to your project, it does not include a project-specific identifier. This may cause issues if another project creates io.github.Foo-Bar which should be its own namespace but areas of flatpak may treat them similar. A better ID would be io.github.foo.Foo even if it is redundant."

This language is not definitive. If there is a hard requirement for what you suggest, I recommend updating the documentation to reflect this.

Reply:
so you want to question the reviewers? Feel free, but the linter will not let it through. The flatpak documentation (that you are quoting) is not flathub's. But clearly say it is problematic, so it is only ambiguous if you want to ignore it.

Reply:
The documentation may not be flathub's, but is linked from https://github.com/flathub/flathub/wiki/App-Requirements. I read the "problematic" language, but also ran into an issue where we cannot have dashes in the 'middle' of the app-id. Since we do not have a site like "obsidian_level_maker.github.io", I decided to err on using a name correlating to a site that is under our purview.